presto-record-decoder/src/test/thrift/tweep.thrift

```diff
@@ -0,0 +1,43 @@
+namespace java com.facebook.presto.decoder.thrift.tweep
+
+enum TweetType {
+    TWEET,
+    RETWEET = 2,
+    DM = 0xa,
```

Comment:
This is just to verify that unconsecutive and different enum values would still work when parsed